### PR TITLE
[Flutter-Parent][MBL-13593][MBL-13652] Alerts list design check

### PR DESCRIPTION
**Pull request by CalvinKern**
MBL-13593: Updated the alerts page to have a title based on the threshold values set.
MBL-13652: Updated empty state to have new asset, title, and subtitle.
Extra: Fixed showing a course announcement that doesn’t have any attachments.

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020434&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020435&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020439&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: apps/flutter_parent/lib/screens/alerts/alerts_screen.dart [hunk 1]
```diff
@@ -89,47 +90,75 @@ class _AlertsScreenState extends State<AlertsScreen> {
...
+                Text(_alertTitle(context, alert), style: textTheme.subtitle.copyWith(color: alertColor)),
+                SizedBox(height: 4),
+                Text(alert.title, style: textTheme.subhead),
+                SizedBox(height: 4),
+                Text(_formatDate(context, alert.actionDate), style: textTheme.subtitle),
+                SizedBox(height: 12),
+              ],
+            ),
+          ),
+          SizedBox(height: 16),
```
Comment (JordanMarshall) [inline apps/flutter_parent/lib/screens/alerts/alerts_screen.dart]: `height` -> `width`? It looks like this was meant to provide horizontal padding.

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: apps/flutter_parent/lib/screens/alerts/alerts_screen.dart (hunk 1 above)
Comment (CalvinKern) [reply apps/flutter_parent/lib/screens/alerts/alerts_screen.dart]: Yuup, nice catch!

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020739&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Pull request merged**